### PR TITLE
Update README.md

### LINKED ISSUE #61: Readme Facelift
When we initially wrote the readme we hadn't fully flushed out the ideas of the two primary components clairvoyance and architect. I think we should add a section to the read that encompasses information about each of the two primary components and maybe includes some of the cool art we had for them.

### PR DESCRIPTION
**Pull request by 0xJepsen**
closes #61

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,18 +3,24 @@
 ![](https://visitor-badge.laobi.icu/badge?page_id=arbiter)
 ![Github Actions](https://github.com/primitivefinance/arbiter/workflows/Rust/badge.svg)
 
-> A stand-alone Rust analytic program for UniswapV3 pools and...
+> A stand-alone analytic program for UniswapV3 pools wirtten in rust.
```
Comment (ts0yu) [inline README.md:6]: minor spelling mistake here

File: README.md [hunk 2]
```diff
@@ -3,18 +3,24 @@
...
 
-We want to develop a Rust program capable of detecting and executing arbitrage opportunities between exchanges. Our goal is to lower the barrier to entry for searching in hopes that it can help level the playing field for capturing arbitrage.
+Searchers detect differences in reported prices between markets and execute pure profit[^1] trades, which we call _atomic arbitrage_. The environment for arbitrage is competitive and it is time to streamline the simluation, and analysis of the implications of arbitrage.
 
 [^1]: 'Pure profit' refers to the fact that atomic arbitrage does not require arbitrageurs to put any capital at risk other than an upfront cost on building the strategy and gas cost to get their transaction included. This contributes to the lucrative nature of atomic arbitrage when done effectively
 
+## Architecture:
+
+There are two primary crates architect, which handles the construction and execution of transactions, and clairvoyance, which handles the monitoring of UniswapV3 pools. The program is designed to be modular and extensible.
```
Comment (ts0yu) [inline README.md:18]: we should add single-line code blocks here to emphasize the fact they are crates - `architect` and `clairvoyance`

**Review by ts0yu — CHANGES_REQUESTED** [review]
few very minor things but otherwise looking good!

File: README.md (hunk 1 above)
Comment (0xJepsen) [reply README.md:6]: nice catch. Spelling is not my strong suit. Hence why i gravitate to math.

File: README.md (hunk 2 above)
Comment (0xJepsen) [reply README.md:18]: nice i'm down

File: README.md (hunk 1 above)
Comment (ts0yu) [reply README.md:6]: love it - no worries

Comment (ts0yu) [reply README.md:6]: love it - no worries

**Review by ts0yu — APPROVED** [review]

**Comment by ts0yu** [comment]
lgtm

**Pull request merged**